src/components/tech/WorkingSection.tsx

```diff
@@ -0,0 +1,65 @@
+import { Flex, Grid, Heading, Section, Text } from "@radix-ui/themes";
+import React from "react";
+import { WorkingBox } from "./WorkingBox";
+
+const data = [
+  {
+    title: "Distribute Aid Website",
+    para: "Since relaunching our website, we have been continually developing new ways to share information and insights from our projects, such as through data visualisations",
+    link: "github.com/distributeaid/next-website-v2",
+    tech: "TypeScript,Next.js,React,Radix UI,Tailwind",
+  },
+  {
+    title: "Needs Assessment",
+    para: "The goal of this project is to provide a way for Distribute Aid to run needs assessments using forms that can be modified using a low-code or zero-code approach",
+    link: "github.com/distributeaid/aggregated-public-information",
+    tech: "Postgres,Typescript,NodeJS,Strapi,Jest,SuperTest",
+  },
+  {
+    title: "Shipment Data Reporting Pipeline",
+    para: "Our Shipment Data Reporting Pipeline ingests data that we record for each shipment, processes it into impact stats, and presents it through a public API that is consumed by our landing site and shared with researchers/other third parties",
+    link: "github.com/distributeaid/aggregated-public-information",
+    tech: "Postgres,Typescript,NodeJS,Strapi,Jest,SuperTest",
+  },
+  {
+    title: "Documentation Platform",
+    para: "We always encourage people to be diligent with documentation. To make it easier for developers to create documentation, we are currently setting up a centralized platform to streamline the documentation process",
+    link: "github.com/distributeaid/docs",
+    tech: "Mintlify",
+  },
+];
+
+export const WorkingSection = () => {
+  return (
+    <Section my="9" mx={{ initial: "5", md: "94px" }} p="0">
+      <Flex gap="4" direction="column" justify="center" align="center">
+        <Heading align="center" className="text-navy-900 text-5xl">
```

Comment:
**Suggestion (non-blocking):** Would set the font size through Radix's`size` propertyon Heading & other typeography elements.  Just choose the nearest size to the design!